util/valitationUtils.js

```diff
@@ -0,0 +1,15 @@
+const removeLightningPrefix = function (invoice) {
```

Comment:
@danfercf1 if it's a new file, please use .ts (typescript)

Reply:
Alright, I can do that